UDF/ExprFunctions.hpp

```diff
@@ -0,0 +1,90 @@
+/******************************************************************************
+ * Copyright (c) 2015-2016, TigerGraph Inc.
```

Comment:
Should update copyright to 2015-2021, because you updated it.
You do not update it just because it's a new year, but whenever you modify the file.